src/app/task-list/task-list.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="list-container">
+  <app-create-task (addTaskEvent)="addTask($event)"></app-create-task>
+
+  <ul *ngIf="tasks$ | async as tasks">
```

Comment:
try also @if and @for, they are the new ways to use these directives in angular, but this is ok as well